internal/cli/cmd/dac/build/testdata_go/go.mod

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+module dac
+
+go 1.21.6
+
+replace github.com/perses/perses => ../../../../../../ // TODO: remove when released
```

Comment:
I don't think we want to remove that after the release since we will always want to test the current implementation and not the previous release